OpenEdXMobile/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -851,9 +857,36 @@ public void onPlaybackComplete() {
                     player.getController().showSpecial( (getTouchExploreEnabled() ? 0L : 5000L) );
                 }
             }
-        }catch(Exception e){
+        } catch(Exception e) {
             logger.error(e);
         }
+        if (environment.getConfig().isAppReviewsEnabled() &&
+                NetworkUtil.isConnected(getContext())) {
+            PrefManager.UserPrefManager userPrefs = new PrefManager.UserPrefManager(MainApplication.application);
+            final float appRating = userPrefs.getAppRating();
+            // If user has not given rating yet, ask for rating
```

Comment:
Yeah it'd be better to define constants for:
- Have not given rating yet.
- Decided not to give rating (if that's possible).